RxExample/RxExample/Examples/GitHubSignup/Views/GitHubSignupViewController.swift

```diff
@@ -108,9 +108,8 @@ class GitHubSignupViewController : ViewController {
                 let validationColor: UIColor
                 
                 if let valid = v.valid {
-                    validationColor = valid ? okColor : errorColor
-                }
-                else {
+                    validationColor = valid ? self.okColor : self.errorColor
+                } else {
```

Comment:
We dispose all subscriptions on `willMoveToParentViewController`, so it's not a leak so far, my bad :)